api/utils/csv.go

```diff
@@ -23,3 +27,42 @@ func ReadCSV(path string) ([][]string, error) {
 	defer csvf.Close()
 	return csvLines, nil
 }
+
+func WriteChangesToCSV(path string, records []model.Contestant) error {
```

Comment:
Would probably rename this to only `WriteCSV` that way we're keeping the naming somehow consistent with the `ReadCSV` func that we already have